src/main/java/com/oya/kr/user/domain/enums/Gender.java

```diff
@@ -0,0 +1,37 @@
+package com.oya.kr.user.domain.enums;
+
+import static com.oya.kr.global.exception.GlobalErrorCodeList.*;
+
+import com.oya.kr.global.exception.ApplicationException;
+
+public enum Gender {
+
+	MAN("man"),
+	WOMAN("woman"),
+	;
+
+	private final String name;
+
+	Gender(String name) {
+		this.name = name;
+	}
+
+	public static Gender getGenderEnum(int code) {
+		switch (code) {
+			case 0:
+				return Gender.MAN;
+			case 1:
+				return Gender.WOMAN;
+		}
+		return null;
+	}
+
+	public static Gender findByName(String name) {
+		for (Gender enumValue : Gender.values()) {
+			if (enumValue.name.equals(name)) {
+				return enumValue;
+			}
+		}
+		throw new ApplicationException(WRONG_ENUM);
+	}
```

Comment:
```suggestion
	public static Gender findByName(String name) {
		return Arrays.stream(values())
                    .filter(gender -> gender.name.equals(name))
                    .findFirst()
                    .orElseThrow(() -> new ApplicationException(WRONG_ENUM));
	}
```

자바의 stream 을 사용하여 멋드러지게 표현하면 어떨까요??